decidim-core/app/uploaders/decidim/image_uploader.rb

```diff
@@ -52,6 +50,15 @@ def max_image_height_or_width
       3840
     end
 
+    def manipulate!
+      begin
+        super
+      rescue CarrierWave::ProcessingError => e
+        STDERR.puts e.inspect
```

Comment:
I'd use rails logger

```suggestion
        Rails.logger.error(e)
```